docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -1117,6 +1117,42 @@ The `quarkus-oidc-token-propagation-reactive` extension provides `io.quarkus.oid
 The `quarkus-oidc-token-propagation-reactive` extension (as opposed to the non-reactive `quarkus-oidc-token-propagation` extension) does not currently support the exchanging or resigning the tokens before the propagation.
 However, these features may be added in the future.
 
+[[oidc-client-graphql-client]]
+== GraphQL client integration
+
+The `quarkus-oidc-client-graphql` extension provides a way to integrate an
+OIDC client into GraphQL clients. This works similarly as with REST clients.
```

Comment:
Please make `GraphQL` or `GrapghQL client` an `xref:` link to the `GraphQL` guide and in reference back to this section from that guide's `References` section for GrapghQL users be aware of this option

Reply:
Done :)